CONTRIBUTING.md

```diff
@@ -86,9 +86,6 @@ vagrant exec bundle exec rspec
 ## Code of Conduct
 OSEM is part of the openSUSE project. We follow all the [openSUSE Guiding Principles!](http://en.opensuse.org/openSUSE:Guiding_principles) If you think someone doesn't do that, please let us know at maintainers@osem.io
 
-## Contact
```

Comment:
In f188fd55 this section was added to REAME.md. Now it is just a duplicate information(with exact same words).

If we are going to keep contact section in contribution.md, can we please use different words?